build.gradle

```diff
@@ -407,6 +407,9 @@ gradle.projectsEvaluated {
       maybeConfigure(project.tasks, 'dependenciesInfo') {
         it.enabled = false
       }
+      maybeConfigure(project.tasks, 'dependenciesGraph') {
```

Comment:
Just a general note, We keep adding more stuff here. In general we should avoid the pattern of adding logic and disabling it to certain projects. Instead we should have a notion of production project or sth similar to have a way to apply logic like this dependencyGraph plugin only on this production projects